src/Symfony/Component/Intl/Resources/emoji/composer.json

```diff
@@ -14,10 +14,9 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "symfony/filesystem": "^6.4|^7.0",
-        "symfony/finder": "^6.4|^7.0",
-        "symfony/var-exporter": "^6.4|^7.0",
+        "symfony/filesystem": "^6.3|^7.0",
+        "symfony/finder": "^6.3|^7.0",
+        "symfony/var-exporter": "^6.3|^7.0",
```

Comment:
Why adding 6.3 here?

Reply:
Or it was because SF 6.4 was not released